Html lang change and readme fixes

Luuwa opened the pull request.

Luuwa committed:
Create main.html

I thought it might make sense to change the HTML lang attribute since we have content mostly in Turkish.

Luuwa committed:
Update README.md

When I tried to run the 'mkdocs publish' command, I got an error that says 'Error: No such command 'publish'.' Then, I checked https://www.mkdocs.org/user-guide/cli/ and found a build command. So
we can change the readme file.

v0lkan commented on overrides/main.html:
```diff
@@ -0,0 +1,2 @@
+<html lang="tr">
+{% extends "base.html" %}
```
I need to fork and test this.

v0lkan replied:
Can we remove this; the final rendition results in

```
<html lang="tr">
<!DOCTYPE html>
<html lang="en">
```

which is invalid markup.

If you want, you can override `base.html`.

v0lkan commented on README.md:
```diff
@@ -15,7 +15,7 @@ The public Website of “Vadideki Geyik” Podcasts — https://geyik.z2h.dev/
 * Install the [terminal theme][terminal].
 * `mkdocs serve` serves the website for local development.
-* `mkdocs publish` will create a production-ready website under the `site` 
+* `mkdocs build` will create a production-ready website under the `site` 
```
Nice catch. Thanks 🙏 .

v0lkan reviewed (changes requested):
One change request in the “overrides” section; the rest looks okay.

Luuwa committed:
Update main.html

I found there is a spesific override path for this https://ntno.github.io/mkdocs-terminal/configuration/blocks/#override-locations

v0lkan commented on overrides/main.html:
```diff
@@ -0,0 +1,2 @@
+{% extends "base.html" %}
+{% block site_lang %}lang="tr"{% endblock %}
```
yup. that appears to sort it out.

thanks for your contribution 🙏 .
